sdk/core/Azure.Core/README.md

```diff
@@ -1,45 +1,100 @@
 # Overview 
 
-Azure.Core is a reusable library used to implement .NET Azure SDK components. 
-It contains APIs that make it easy to call Azure service endpoints in a convinient, performant, and consistent manner.
+Azure.Core provides shared primitives, abstractions, and helpers for .NET Azure SDK service client libraries. 
```

Comment:
nit:  What's a "helper"?    That seems nebulous enough to have different meanings to different developers.

Reply:
Is there any software term that does not have different meanings to different developers? :-)
For example, does the term "primitive" not have this problem?

But, if you have a suggestion how to fix it (reword it), I would be more than happy to.